### PR TITLE
Documentation Fixes

### PR DESCRIPTION
**Pull request by Airblader**

### CODE REVIEW

File: downloads/index.html [hunk 1]
```diff
@@ -358,26 +358,23 @@
...
   The 'next' branch is the next version, yet to be released. Use <tt>git
-  checkout next</tt> to switch to the 'next' branch.
+  checkout master</tt> to switch to the 'master' branch.
 </p>
 
 <p>
   If you prefer to download a tarball, or if you cannot use git for whatever reason,
-  you may download the current master branch from
-  <a href="http://code.i3wm.org/i3/snapshot/i3-master.tar.bz2">
-    http://code.i3wm.org/i3/snapshot/i3-master.tar.bz2
```
Comment (Airblader) [inline downloads/index.html]: This tarball still contains 4.9.

Comment (stapelberg) [inline downloads/index.html]: I’ve fixed that — hopefully for real now :). Mirroring git repositories is harder than I thought.

File: downloads/index.html [hunk 2]
```diff
@@ -358,26 +358,23 @@
...
-  </a>
+  you can find individual releases <a href="https://github.com/i3/i3/releases">here</a>.
```
Comment (stapelberg) [inline downloads/index.html]: This is not a good substitute. Please use https://github.com/i3/i3/archive/master.tar.gz instead, which is the corresponding github link. The link gives you a snapshot of master, not just the released versions.

**Comment by Airblader** [comment]
@stapelberg Done

File: downloads/index.html [hunk 3]
```diff
@@ -358,26 +358,26 @@
 
 <p>
   Use <a href="http://git-scm.com/">git</a> to follow the latest changes or
-  <a href="http://code.i3wm.org/i3">view them on this webpage</a>. This command checks out the 'master'
+  <a href="https://www.github.com/i3/i3">view them on this webpage</a>. This command checks out the 'next'
```
Comment (stapelberg) [inline downloads/index.html]: Please remove the www.

**Comment by Airblader** [comment]
@stapelberg Done. Out of curiosity – why?

**Comment by stapelberg** [comment]
Just so that it’s consistent with all the other links we use. Also, if you look closely, you’ll see that www.github.com redirects to github.com, so by linking to the target of that redirection, you save one round trip.

**Comment by Airblader** [comment]
Indeed. Thanks for the explanation.

**Pull request merged**